src/main/java/com/unosquare/adminCore/entity/Holiday.java

```diff
@@ -0,0 +1,59 @@
+package com.unosquare.adminCore.entity;
+
+import com.fasterxml.jackson.annotation.JsonIdentityInfo;
+import com.fasterxml.jackson.annotation.ObjectIdGenerators;
+import lombok.Data;
+
+import javax.persistence.*;
+import java.time.LocalDate;
+
+@Entity
+@Data
+@JsonIdentityInfo(generator = ObjectIdGenerators.PropertyGenerator.class, property = "holidayId")
+
+@Table(name = "Holiday")
+public class Holiday {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int holidayId;
+
+    private LocalDate startDate;
+    private LocalDate endDate;
+
+    private short holidayLength;
```

Comment:
Is holiday length needed? Having something like this places a requirement on having to keep it up to date and as we add more criteria in it will require tasks to keep in current.

I'd keep it to the date range only for now, performance wouldn't be an issue.